test/Microsoft.AspNetCore.SignalR.Client.Tests/HttpConnectionTests.Negotiate.cs

```diff
@@ -37,17 +37,16 @@ public Task StartThrowsFormatExceptionIfNegotiationResponseHasNoConnectionId()
             }
 
             [Fact]
-            public Task StartThrowsFormatExceptionIfNegotiationResponseHasNoTransports()
+            public Task ConnectionCannotBeStartedIfNoCommonTransportsBetweenClientAndServer()
             {
-                return RunInvalidNegotiateResponseTest<InvalidOperationException>(ResponseUtils.CreateNegotiationContent(transportTypes: 0), "Unable to connect to the server with any of the available transports.");
+                return RunInvalidNegotiateResponseTest<AggregateException>(ResponseUtils.CreateNegotiationContent(transportTypes: HttpTransportType.ServerSentEvents),
+                    $"Unable to connect to the server with any of the available transports. (ServerSentEvents is disabled by the client.)");
```

Comment:
not using interpolation?

```suggestion
                    "Unable to connect to the server with any of the available transports. (ServerSentEvents is disabled by the client.)");
```